### DISCUSSION TITLE
KAMP travel speed/acceleration settings

### DISCUSSION THREAD

**Discussion by kemsky** [Guilouz/Creality-Helper-Script-Wiki#469] [Issues]
It seems that KAMP uses speed/accel values that are too high, lowering machine limits does not help. I installed less powerful 0.9 motors on my K1 and they can't handle speed/accel, start skipping steps.

I'm no expert but I found this line in `Smart_Park.cfg`:

```
set travel_speed = (printer.toolhead.max_velocity) * 60 | float
```

Why `max_velocity` is multiplied by 60?

**Comment by kemsky** [comment]
ok, `travel_speed` must be in mm/min but in config we have mm/sec so it is multiplied by 60, but still acceleration seems to be too high.

**Reply by VORONENTHUSIAST** [reply]
Hi, Kemsky - after getting frustrated with this, I tinkered around in the KAMPS folder by using WINSCP - you can edit the files in there.
You can see from the below screen grab that my change is now implemented when viewing the file in fluid. 
Completely forgot about this last night - it's been a while since I've messed around with my printers.

Hope this helps!
  
![Line_Purge](https://github.com/user-attachments/assets/8b73c9c1-14c9-405e-af18-30a3bfd84fa3)

**Reply by VORONENTHUSIAST** [reply]
As an FYI, this was still bugging my motors out - so I changed the travel speed to 12000 (200mm/s) and it's fine now. 
![KAMPS](https://github.com/user-attachments/assets/f0d5fa0e-ec12-45da-949c-c5c836bf1639)

**Reply by kemsky** [reply]
@VORONENTHUSIAST, I did similar thing, changed speed manually.

**Comment by VORONENTHUSIAST** [comment]
I'm also finding this to be a problem, if I turn Line Purge off it moves to the print position just fine, with it on it bugs my motors out and causes them to loose steps making the print start in the wrong position and crashing my toolhead. Please make it editable 

**Edit** Totally forgot you're able to edit these files in WinSCP!!

**Reply by mattfox27** [reply]
what file is this?  My kamp goes to fast after purge
